src/main/java/net/sf/jabref/gui/actions/OpenBrowserAction.java

```diff
@@ -0,0 +1,49 @@
+package net.sf.jabref.gui.actions;
+
+import java.awt.event.ActionEvent;
+
+import javax.swing.AbstractAction;
+import javax.swing.Action;
+import javax.swing.Icon;
+
+import net.sf.jabref.gui.desktop.JabRefDesktop;
+
+public class OpenBrowserAction extends AbstractAction {
```

Comment:
No. Only the `JabRefDesktop.openBrowserShowPopup(...)` method is used there - but not by an `AbstractAction` (which is used for menus and icon bars only).